### PR TITLE
Update helm chart script

### LINKED ISSUE #926: Automate helm chart updates
**What would you like to be cleaned**:
Ref: https://github.com/kubernetes-sigs/kueue/pull/925#issuecomment-1613062055
**Why is this needed**:

### PR DESCRIPTION
**Pull request by stuton**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
It helps to sync files from RBAC folder

#### Which issue(s) this PR fixes:

Fixes #926

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: charts/kueue/templates/rbac/auth_proxy_role.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  verbs:
-  - create
-- apiGroups:
-  - authorization.k8s.io
-  resources:
-  - subjectaccessreviews
-  verbs:
-  - create
+  - apiGroups:
+      - authentication.k8s.io
```
Comment (stuton) [inline charts/kueue/templates/rbac/auth_proxy_role.yaml:7]: There is an [issue](https://github.com/mikefarah/yq/issues/825) regarding indents which **yq** makes. The flag -I0 doesn't work properly, that's mean ignore indent. Once the problem problem is solved, I will return the indents back.

File: config/components/rbac/rayjob_editor_role.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: kueue-rayjob-editor-role
```
Comment (stuton) [inline config/components/rbac/rayjob_editor_role.yaml:5]: Let's keep the naming the same as MPIJob

**Comment by alculquicondor** [comment]
/retitle Automatically sync RBAC files in helm chart

**Pull request merged**